mentoring-app/src/app/footer/footer.component.scss

```diff
@@ -0,0 +1,7 @@
+.footer {
+    background: white;
+
+    &_text {
+        font-size: 16px;
+    }
+}
```

Comment:
According to the homework requirements, your footer should be sticky, i.e. always stay at the bottom of the window

Reply:
i gave min-height: 100% to the main content, so it should alway be at the bottom